python/res/job_queue/queue.py

```diff
@@ -210,6 +211,9 @@ def get_max_job_duration(self):
     def set_max_job_duration(self, max_duration):
         self._set_max_job_duration(max_duration)
 
+    def get_max_submit(self):
```

Comment:
Property?

Reply:
`max_submit` property now resides in queue only and `queue_manager` pokes queue for it, so `max_submit` function is gone from `queue_manager`